test/libp2p/protocol/echo_test.cpp

```diff
@@ -66,16 +71,16 @@ TEST(EchoTest, Server) {
  * @when client writes string "hello" to the Stream
  * @then client reads back the same string
  */
-TEST(EchoTest, Client) {
+TEST(EchoTest, DISABLED_Client) {
```

Comment:
Comment, please, why is it disabled and issue (number or link) to enable it back. 
Or remove if it really needless anymore

Reply:
this test expect reentrant behavior from mocks, needs to be rewritten as soon as new testing tools for async appears (net PR)